cli/src/alluxio.org/cli/cmd/conf/log.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package conf
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/env"
+)
+
+var Log = &LogCommand{
+	BaseCommand: &env.BaseCommand{
+		CommandName:   "log",
+		JavaClassName: "alluxio.cli.LogLevel",
+	},
+}
+
+type LogCommand struct {
+	*env.BaseCommand
+
+	LogName string
+	Level   string
+	Target  string
+}
+
+func (c *LogCommand) Base() *env.BaseCommand {
+	return c.BaseCommand
+}
+
+func (c *LogCommand) ToCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("%v --name <name> [--level <level>] [--target <target>]", Log.CommandName),
```

Comment:
ditto to my previous comment, I wish the `Use` argument could be auto-generated based on the flags the command registers.